test/test_logging_console_output_handler.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <vector>
+
+#include "osrf_testing_tools_cpp/scope_exit.hpp"
+#include "rcutils/logging.h"
+
+static void call_handler(
+  const rcutils_log_location_t * location,
+  int severity, const char * name, rcutils_time_point_value_t timestamp,
+  const char * format, ...)
+{
+  va_list args;
+  va_start(args, format);
+  rcutils_logging_console_output_handler(location, severity, name, timestamp, format, &args);
+  va_end(args);
+}
+
+TEST(TestLoggingConsoleOutputHandler, typical_inputs) {
+  ASSERT_EQ(RCUTILS_RET_OK, rcutils_logging_initialize());
+  OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT(
+  {
+    EXPECT_EQ(RCUTILS_RET_OK, rcutils_logging_shutdown());
+  });
+
+  rcutils_log_location_t log_location = {
```

Comment:
@brawner what follows seems akin to a smoke test to me. Shall we leave a comment stating so? Same for the other test case below.

Reply:
Added a more informative comment at the top of the test function.

Reply:
Cool. Mocks would come in handy here...